tutorials/xr/ar_passthrough.rst

```diff
@@ -0,0 +1,187 @@
+.. _doc_openxr_passthrough:
+
+AR / Passthrough
+================
+
+Augmented Reality is supported through various methods depending on the capabilities of the hardware.
+
+Headsets such as the Magic Leap and glasses such as TiltFive show the rendered result on transparent displays
```

Comment:
We can add that the technique is usually referred to as [see-through] (https://en.wikipedia.org/wiki/See-through_display).

Reply:
owh I like that term :)